Automattic/jetpack
# Blocks: avoid invalid category warnings in editor

## Proposed changes:

Our template used the old 'jetpack' category, which is no longer an option. I've updated that and updated the 2 recent blocks that used that wrong category, to avoid the following warning in the editor:

```
The block "jetpack/blogging-prompt" is registered with an invalid category "jetpack".
```

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Make sure beta blocks are enabled on your site:

```php
add_filter( 'jetpack_blocks_variation', function () {
	return 'beta';
} );
```

* Go to Posts > Add New
* Do you see any notices in your browser console?
